Project.html

```diff
@@ -0,0 +1,131 @@
+                <option value="saudiarabia">Saudi Arabia</option>
+            </select>
+        </span>
+
+        <span class="card-2">
+            <img src="https://www.docplanner.com/icons/icon-doctors.svg">
+            <h3>For Doctors</h3>
+            <p>Save time managing visits and cut no-shows by half</p>
+            <select class="drop-menu">
+                <option value="choosecountry">Choose Country</option>
+                <option value="Argentina">Argentina</option>
+                <option value="Brazil">Brazil</option>
+                <option value="Egypt">Egypt</option>
+                <option value="Qatar">Qatary</option>
+                <option value="uae">UAE</option>
+                <option value="usa">USA</option>
+                <option value="saudiarabia">Saudi Arabia</option>
+            </select>
+        </span>
+
+        <span class="card-3">
+            <img src="https://www.docplanner.com/icons/icon-clinics.svg">
+            <h3>For Clinics</h3>
+            <p>Deliver an exceptional patient experience in your clinic</p>
+        </span>
+    </div>
+
+    <div class="info">
+        <span class="fact">
+            <h1>The world's biggest healthcare platform</h3>
+            <p>We work from 9 offices all over the world, bringing Docplanner Group to life in 13 countries.</p>
+        </span>
+        <!-- <div class="main-div"> -->
+            <span class="branches">
+                <img class="logo" scr="https://www.docplanner.com/img/flag.png" srcset="https://www.docplanner.com/img/flag.png 1x, https://www.docplanner.com/img/flag@2x.png 2x">
+                <h4>Leader in 13 countries</h3>
+                <p>Poland, Brazil, Mexico, Spain, Italy, Germany, Turkey, Colombia, Czech Republic, Portugal, Argentina, Peru and Chile</p>
+            </span>
+
+            <span class="stats">
+                <img class="logo" scr="https://www.docplanner.com/img/visits.png" srcset="https://www.docplanner.com/img/visits.png 1x, https://www.docplanner.com/img/visits@2x.png 2x">
+                <h4>15,000,000 appointments </h3>
+                <p>booked last months</p>
+            </span>
+
+            <span class="patients">
+                <img class="logo" scr="https://www.docplanner.com/img/patients.png" srcset="https://www.docplanner.com/img/patients.png 1x, https://www.docplanner.com/img/patients@2x.png 2x">
+                <h4>90,000,000 patients</h3>
+                <p>visit our websites each month</p>
+            </span>
+
+            <span class="doctors">
+                <img class="logo" scr="https://www.docplanner.com/img/doctors.png" srcset="https://www.docplanner.com/img/doctors.png 1x, https://www.docplanner.com/img/doctors@2x.png 2x">
+                <h4>210,000 active doctors</h3>
+                <p>trust our solutions</p>
+            </span>
+        <!-- </div> -->
+    </div>
+    <footer>
+        <p>We are leaders in 13 countries:
+            <a href="">Poland</a>
```

Comment:
I think it would have better (and would making styling easier) if all these anchor elements are grouped together.